pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "sentencepiece",
     "tiktoken",
     "blobfile>=2",
+    "tokenizers",
```

Comment:
I had thought we would make this a dev dependency and conditionally import this in hf_tokenizer

Reply:
Yeah I was on the fence about this. But actually I think the upstream dependencies of tokenizers are already a subset of our upstream dependencies. So in that case it's not like we are pulling in a bunch of new transitive deps, this is really just tokenizers and nothing else. Because of that I feel OK about having it as a core dep, but happy to take the approach you're suggesting if you (or others) strongly disagree

Reply:
Just documenting. Tokenizer package is roughly 3 MB (https://pypi.org/project/tokenizers/#files)

Reply:
Isn't torch close to 1 GB? Anyways if everyone would prefer to make this an optional dep I am fine with it

Reply:
Yeah, 3 MB is nothing. I'm fine with it as a core dep.